packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss

```diff
@@ -90,7 +90,8 @@
 
   padding-left: units($theme-alert-icon-size) + ($alert-icon-optical-padding);
   @include at-media($theme-site-margins-breakpoint) {
-    @include u-padding-x($theme-site-margins-width * 2);
+    padding-left: units($theme-site-margins-width) * 2;
+    padding-right: units($theme-site-margins-width) * 2;
```

Comment:
I avoided padding-inline here because it felt like it had meaning beyond what we intend at this time. I like the idea of moving towards `padding-inline`/`padding-block` in the future to better accommodate different language types, but in my mind it would require a comprehensive effort where we write styles for both block and inline. I might be overthinking it though! Curious to hear thoughts.

Reply:
I think that's a valid point! Not opposed to leaving it as is since it is more specifically defined.

Reply:
I'm happy to now know about these two new properties

Reply:
In this case it's the same value on each side, which at least makes it equivalent in LTR or RTL reading directions....